### DISCUSSION TITLE
Схемы и иллюстрации в тексте Откровения.

### DISCUSSION THREAD

**Discussion by irinachernov** [tigran123/ud#3] [Идеи и вопросы]
Как вы знаете, у нас в проекте есть первый рисунок: "Семь абсолютов и структура реальности".

Вопрос ко всем участникам проекта: Есть ли какие-либо идеи оносительно схем, рисунков, иллюстраций, которые можно было бы воплотить в дальнейшем в тексте Откровения? Если возникнут идеи и желание поделиться, будет интересно попробовать их воплотить. 

Спасибо всем!

**Comment by tigran123** [comment]
Я думаю, что по мере перевода текста мы будем включать, как минимум, все те иллюстрации, которые имеются в Британском учебном издании (см. https://github.com/tigran123/urantia-study-edition/tree/master/images). Конечно, можно и нужно добавлять иллюстрации новые, которых нет там. Но если новых идей пока нет, то следует взять какую-нибудь из иллюстраций оттуда (например https://github.com/tigran123/urantia-study-edition/blob/master/images/TheOceansOfSpace-small.jpg) и перевести на русский, или создать русский эквивалент -- лучше и красивее.

**Reply by irinachernov** [reply]
Спасибо, это очень интересно и красиво!

**Comment by 1UnionStation** [comment]
> Как вы знаете, у нас в проекте есть первый рисунок: "Семь абсолютов и структура реальности".
> 
> Вопрос ко всем участникам проекта: Есть ли какие-либо идеи оносительно схем, рисунков, иллюстраций, которые можно было бы воплотить в дальнейшем в тексте Откровения? Если возникнут идеи и желание поделиться, будет интересно попробовать их воплотить.
> 
> Спасибо всем!

Доброго времени суток, ниже прикрепляю видео, возможно видео в чем-то поможет https://www.youtube.com/watch?v=1XXZp26g3YA&feature=emb_logo

**Reply by irinachernov** [reply]
1UnionStation - Спасибо большое за помощь! Это замечательное наглядное видео!

**Reply by alyuev** [reply]
Это же видео, только с переводом - https://www.youtube.com/watch?v=EZ-JRhWhA6o&list=PLTI6ZOS7Dg7Pm21etyaF9q_KMzye5NKI6

**Reply by irinachernov** [reply]
Спасибо, с переводом ещё понятней и наглядней!

**Reply by tigran123** [reply]
Спасибо, конечно, но, когда я смотрел оригинал этого ролика лет 5 назад, я не нашёл в нём ничего интересного или полезного.
